gdraw/ggdkdraw.c

```diff
@@ -2557,6 +2593,17 @@ G_GNUC_END_IGNORE_DEPRECATIONS
 
     _GDraw_InitError((GDisplay *) gdisp);
 
+#ifdef FONTFORGE_CAN_USE_GTK_BRIDGE
+    // XXX These may be preferable to the settings in
+    // _GGDKDraw_CreateWindow, but using both should be OK
+    // as long as the same strings are used
+    gdk_set_program_class(GResourceProgramName);
+    // GTK derives the WM_CLASS from this
+    g_set_prgname(GResourceProgramName);
+
+    gdisp->gtkb_state = gtkb_CreateState();
+#endif
```

Comment:
This makes icons and task bar elements in X group with FontForge elements. I put these here because they're gobject and gdk level rather than strictly gtk.